examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/ValidationExceptionHandler.java

```diff
@@ -0,0 +1,94 @@
+package example.springframework.boot.webflux;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.Instant;
+
+import javax.validation.ValidationException;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
+
+/**
+ * A sample exception handler which handles a {@link ValidationException}.
+ */
+public class ValidationExceptionHandler implements ExceptionHandlerFunction {
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public HttpResponse handleException(RequestContext ctx, HttpRequest req, Throwable cause) {
+        if (cause instanceof ValidationException) {
+            final HttpStatus status = HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
Perhaps `BAD_REQUEST`? We could use `INTERNAL_SERVER_ERROR` on `JsonProcessingException` though.